### PR TITLE
Fix pyright

### PR DESCRIPTION
**Pull request by crsegerie**

### CODE REVIEW

File: kili/types.py
Comment (PierreLeveau) [inline kili/types.py:14]: separate the replacement of dataclasses by the typedicts

File: requirements.txt
Comment (PierreLeveau) [inline requirements.txt:8]: ☝️ rm [recipes/download_google_drive](https://github.com/kili-technology/kili-python-sdk/pull/657/files#diff-aceaba362149f189bc98fbfe0646e9142bf339484a4c506985ac00e1b1de6dd9)

File: requirements.txt
Comment (PierreLeveau) [inline requirements.txt:11]: rm from baseline requirements

File: test/services/export/fakes/fake_kili.py
Comment (PierreLeveau) [inline test/services/export/fakes/fake_kili.py:5]: rm # type:ignore

File: test/test_client_project.py
Comment (PierreLeveau) [inline test/test_client_project.py:37]: rm need to transtype

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: kili/client.py
Comment (crsegerie) [reply kili/client.py:30]: in fact, It does not make sense to make this change considering the content of typing.py

**Review by PierreLeveau — APPROVED** [review]
LGTM, but wait until the release is done before merge

**Pull request merged**
